### DISCUSSION TITLE
Is there any proper way to delete a workspace.

### DISCUSSION THREAD

**Discussion by leftover19** [vercel/turborepo#7893] [Help]
### Summary

I created an app named CloudFrontDeploy and after some coding, which include installing packages, I deleted that folder and created new app with name AWSDeployer. 
But I can see CloudFrontDeploy in package-lock.json

        `"apps/CloudFrontDeploy": {
            "name": "cloudfrontdeploy",
            "extraneous": true,
            "dependencies": {
                "@types/express": "^4.17.21",
                "aws-sdk": "^2.1590.0",
                "cors": "^2.8.5",
                "express": "^4.19.2",
                "redis": "^4.6.13",
                "simple-git": "^3.24.0"
            },
            "devDependencies": {
                "bun-types": "latest",
                "nodemon": "^3.1.0",
                "typescript": "^5.4.3"
            },
            "peerDependencies": {
                "typescript": "^5.0.0"
            }
        },

1. Is this app not deleted properly?  If not, how to delete this  ?

1. If a again create an app named CloudFrontDeploy, I suppose these dependencies will be available for it. 

3. What if I rename an app. Will it create new "app/<new name>" or will it replace existing "app/

**Comment by anthonyshew** [comment]
Have you regenerated the lockfile using your package manager's install command? You may also need to delete the lockfile before the install command to clear it up.
